.github/workflows/release.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
         sudo apt update && sudo apt install -y jq
     - id: set-matrix
       run: |
-          content=`cat ./.github/flavors.json | jq -r 'map(select(.arch == "amd64" and .variant == "core" and (.flavor == "fedora" or (.flavor == "ubuntu" and .flavorRelease == "23.10"))))'`
+          content=`cat ./.github/flavors.json | jq -r 'map(select(.arch == "amd64" and .variant == "core" and (.flavor == "fedora" or (.flavor == "ubuntu" and (.flavorRelease == "23.10" or .flavorRelease == "24.04")))))'`
```

Comment:
I thought it might be good to also produce the 24.04 uki in case it's helpful to test that one too